client/my-sites/email/email-providers-in-depth-comparison/style.scss

```diff
@@ -1,5 +1,35 @@
+@import '@automattic/typography/styles/variables';
+
 .email-providers-in-depth-comparison__header {
 	font-size: xxx-large;
 	margin-bottom: 64px;
 	text-align: center;
+
+	@extend .wp-brand-font;
+}
+
+.email-providers-in-depth-comparison-table {
+	display: flex;
+	gap: 20px;
+}
+
+.email-providers-in-depth-comparison-table__provider-name {
+	display: flex;
+
+	h2 {
+		font-size: $font-title-medium;
+		margin-left: 5px;
+
+		@extend .wp-brand-font;
+	}
+
+	.google-workspace {
+		width: 48px;
+	}
+
+	.professional-email {
+		color: var( --color-wordpress-com );
+		height: 48px;
+		width: 48px;
+	}
```

Comment:
Should these not have a `.logo` or similar class to make it clearer what they are?

Reply:
I'm not sure about adding a separate class but I agree that we could use more descriptive class names. I'll update them.